application-json/src/main/java/ru/art/json/descriptor/JsonEntityWriter.java

```diff
@@ -22,7 +22,7 @@
 import lombok.experimental.*;
 import ru.art.core.checker.*;
 import ru.art.entity.*;
-import ru.art.entity.constants.ValueType;
+import ru.art.entity.constants.*;
```

Comment:
our static analysis engine detects that your pull request includes a change from an explicit import to a wildcard import. According to the best practice (http://www.javadude.com/articles/importondemandisevil.html), import star (import on demand) is a bad practice, wildcard import can sometimes cause name conflicts and ambiguities.